Assets/Scripts/Main Menu/Main Menu Buttons.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
```

Comment:
Would you mind refactoring to use the [Unity Canvas](https://docs.unity3d.com/Manual/UICanvas.html) system?